frontend/src/entry/settings/ManualEntryAccess.tsx

```diff
@@ -0,0 +1,67 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { Accordion, AccordionDetails, AccordionSummary, Button, Stack, TextField, Typography } from '@mui/material'
+import { getUserInformation } from 'actions/user'
+import { useState } from 'react'
+import MessageAlert from 'src/MessageAlert'
+import { CollaboratorEntry, EntityKind } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+
+interface ManualEntryAccessProps {
+  accessList: CollaboratorEntry[]
+  setAccessList: (accessList: CollaboratorEntry[]) => void
+}
+
+export default function ManualEntryAccess({ accessList, setAccessList }: ManualEntryAccessProps) {
```

Comment:
Consider a name change to something like `ManualEntityInput` as this component isn't responsible for managing entry access in itself.